examples/WritingSALScript.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<p>the issue here is that the take_object command will result in an image being taken while the telescope is offsetting when changing gratings/filters....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>